channeldb/graph.go

```diff
@@ -2796,7 +2833,10 @@ func putChanEdgePolicy(edges *bolt.Bucket, edge *ChannelEdgePolicy, from, to []b
 
 	// If there was already an entry for this edge, then we'll need to
 	// delete the old one to ensure we don't leave around any after-images.
-	if edgeBytes := edges.Get(edgeKey[:]); edgeBytes != nil {
+	// An unknown policy value does not have a update time recorded, so
+	// it also does not need to be removed.
+	if edgeBytes := edges.Get(edgeKey[:]); 
```

Comment:
This change not needed anymore?

Reply:
Still needed now that we store unknown policy values.

Reply:
It was like that, but I prefer just checking for an empty slice. Yes, now it cannot be any byte sequence anymore, but I think yagni. Not very important, if you want I'll revert it.